src/resources/views/crud/buttons/create.blade.php

```diff
@@ -1,5 +1,5 @@
 @if ($crud->hasAccess('create'))
-    <a href="{{ url($crud->route.'/create') }}" class="btn btn-primary" data-style="zoom-in">
-        <span><i class="la la-plus"></i> {{ trans('backpack::crud.add') }} {{ $crud->entity_name }}</span>
+    <a href="{{ url($crud->route.'/create') }}" class="btn btn-primary" bp-button="create" data-style="zoom-in">
```

Comment:
I wouldn't remove the `bp-button` - maybe someone's using it for something custom.